scripts/build_torch_wheels.sh

```diff
@@ -86,14 +86,17 @@ function maybe_install_sources {
     popd
   fi
 
-  # Check if we have cloned pytorch and cd into the pytorch dir. Within the pytorch
-  # dir there is a subdir `torch`.
-  if [ ! -d "torch" ]; then
+  # Check if we have cloned pytorch or xla and cd into the pytorch dir. 
+  # Within the pytorch dir there is a subdir `torch`, and within xla 
+  # is `torch_xla`.
+  if [ ! -d "torch" && ! -d "torch_xla" ]; then
     sudo apt-get install -y git
     git clone --recursive https://github.com/pytorch/pytorch.git
     cd pytorch
     git clone --recursive https://github.com/pytorch/xla.git
     export RELEASE_VERSION="nightly"
+  elif [ -d "torch_xla" ]; then
```

Comment:
do you mind adding a comment about under what circumstance there will be a `torch_xla` dir?

Reply:
if you git clone PyTorch/XLA repo, then the project should contain `torch_xla` dir. So the short answer, if you have cloned the repo. Just like checking for `torch` dir, I think this `torch_xla` check can be implied IMO.

Reply:
yeah I know, I guess what I want to know is under what occasion we have done a `git clone pytorch/xla repo`.